doc/classes/Object.xml

```diff
@@ -185,34 +191,35 @@
 			<argument index="0" name="name" type="String">
 			</argument>
 			<description>
-				Return a metadata from the object.
+				Returns [code]metadata[/code] from the Object.
```

Comment:
Returns the given [code]name[/code] from the object's metadata.

Reply:
I think this is the only correction where I put something different than what you have.

I put:
Returns the object's metadata for the given [code]name[/code].

... Since it isn't returning a name, but is returning metadata.